scripts/prepdocs.py

```diff
@@ -388,28 +364,24 @@ async def main(strategy: Strategy, credential: AsyncTokenCredential, args: Any):
         required=False,
         help="Optional, required if --searchimages is specified. Endpoint of Azure AI Vision service to use when embedding images.",
     )
-    parser.add_argument(
-        "--visionkey",
-        required=False,
-        help="Required if --searchimages is specified. Use this Azure AI Vision key instead of the instead of the current user identity to login.",
-    )
     parser.add_argument(
         "--keyvaultname",
         required=False,
         help="Required only if any keys must be fetched from the key vault.",
     )
-    parser.add_argument(
-        "--visionsecretname",
-        required=False,
-        help="Required if --searchimages is specified and --keyvaultname is provided. Fetch the Azure AI Vision key from this key vault instead of using the current user identity to login.",
-    )
     parser.add_argument(
         "--useintvectorization",
         required=False,
         help="Required if --useintvectorization is specified. Enable Integrated vectorizer indexer support which is in preview)",
     )
     parser.add_argument("--verbose", "-v", action="store_true", help="Verbose output")
     args = parser.parse_args()
+
+    if args.verbose:
+        # set formatter to only show the message
```

Comment:
should we comment to state this is for the SDK workaround as discussed?

Reply:
Good idea, adding